docs/client-script-usage.md

```diff
@@ -36,7 +36,7 @@ Notes:
 2. System names can only contain lowercase letters, numbers, and hyphens.
 3. Typically, you would use the `--publish` option when committing to the main/master branch, and you would *not* use it for pull requests.  
 4. These files and directories are excluded *on top of* Sigrid's default excludes. By default, Sigrid excludes things like third party libraries (e.g. `/node_modules/` for NPM libraries, build output (e.g. `/target/` for Maven builds), and generated code. 
-5. The `--pathprefix` option can be used in cases where your repository used a different directory structure from the one that is known to Sigrid. For example, you might have combined your back-end and front-end repositories within a single system in Sigrid, so that in Sigrid there are two top-level folders: `backend` and `frontend` containing the contents of your two repositories. However, you still want to get specific feedback for your front-end repository in Sigrid CI. In this case you would use `--pathprefix frontend` so that Sigrid CI knows the location of your repository within the larger directory structure. Note that you cannot use this option if you're already using `--publish`.  
+5. The `--subsystem` option can be used in cases where your repository is part of a group of repositories that are a single system in Sigrid. For example, you might have combined your back-end and front-end repositories within a single system in Sigrid, so that in Sigrid there are two top-level folders: `backend` and `frontend` containing the contents of your two repositories. However, you still want to get specific feedback for your front-end repository in Sigrid CI. In this case you would use `--subsystem frontend` so that Sigrid CI knows the location of your repository within the larger system.
```

Comment:
I propose: 
"The `--subsystem` option can be used when your repository is part of a group of repositories that is considered as being a single system in Sigrid.

For example, you might have combined your back-end and front-end repositories into a single system in Sigrid, such that in Sigrid there are two top-level folders: `backend` and `frontend` containing the contents of both repositories. However, you still want to get specific feedback for your `frontend` repository in Sigrid CI. In this case, you would use `--subsystem frontend` so that Sigrid CI knows the location of your repository within the larger system."